Include minizip's headers in the nuget pkg

BillHoag opened the pull request:
* make minizip's headers available to consumers of the zlib nuget pkg
* add param iMatchJustFileName to unzLocateFile ()

David-Defoort commented on Zlib.autopkg:
```diff
@@ -22,18 +22,22 @@ nuget
    {
-      nestedInclude: { #destination=${d_include}Zlib; *.h };
+      nested2Include: { #destination=${d_include}Zlib; "*.h" };
+      nested1Include: { #destination=${d_include}contrib\minizip; "*.h" };
```
This is not correct. It means that all headers should go under `contrib\minizip`
![image](https://user-images.githubusercontent.com/5627161/121660545-a7492200-caa3-11eb-9af5-8b5f75781a12.png)
(BTW you can open up the nuget package with 7zip, no need to install the package)

David-Defoort replied:
I think it should be something like (untested):
```
{ #destination=${d_include}Zlib\contrib\minizip; "contrib\minizip\*.h" };
                           ^
```
We want the contrib to go under Zlib, so that the consumer does:
`#include <ZLib/contrib/minizip/ioapi.h>`
instead of
`#include <contrib/minizip/ioapi.h>` <<<< not Zlib specific, it does not look good.

BillHoag replied:
🤦 I thought I saw the correct ones there before - I'm sorry waste your time with this one. Re 7zip, yeah, I've got *.nupkg associated with 7zip. Fixed nupkg is [here](https://drive.google.com/drive/folders/1kvNZMQmgX5vOm0FutWcAeb-4ew8qwcN5?usp=sharing). Corrected autopkg has been force pushed.

David-Defoort replied:
Thanks for the update and np on me trying out that version. Will test it on Monday!
Was it built on your machine?

One thing that I still have to look into is that pesky warning:
`warning LNK4098: defaultlib 'libcmt.lib' conflicts with use of other libs; use /NODEFAULTLIB:library`
Not exactly sure what project is "at fault" and how I can solve this but I will look on Monday

BillHoag replied:
Yes, built on my machine. Time permitting, I'll try your branch and this nupkg on my branch today, and see if I can learn anything about that warning.

David-Defoort reviewed (changes requested):
The resulting nuget package does not have the files that we need `ioapi.h` and `unzip.h`

David-Defoort commented on Zlib.autopkg:
```diff
@@ -22,18 +22,22 @@ nuget
    {
-      nestedInclude: { #destination=${d_include}Zlib; *.h };
+      nested2Include: { #destination=${d_include}Zlib; "*.h" };
+      nested1Include: { #destination=${d_include}Zlib\contrib\minizip; "contrib\minizip\*.h" };
```
Looks good. Will test on Monday :)

Commit:
Use relative paths for zlib includes in minizip

This is necessary so the minizip headers will be found by the compiler and
successfully compiled in SnagitWin when using the zlib NuGet pkg.

Commit:
Include minizip's headers in the nuget pkg

Commit:
Rev version to 1.2.11.3

David-Defoort commented:
It all looks good. Used the nuget package that @BillHoag built locally and confirm this would work just fine in SnagitWin 👍
